app/assets/javascripts/app/checkbox_form.js

```diff
@@ -1,6 +1,6 @@
 $(document).ready(function() {
   let submitButton = $(".js--requireValueForSubmit");
-  let checkboxes = submitButton.parents("form").find(":checkbox");
+  let checkboxes = submitButton.parents("form").find(":checkbox").not('[name="bulk_note_checkbox"]');
```

Comment:
What do you think about using a new class for the checkboxes for the rows, and then use that selector? I find it clearer than selecting them all and then exclude one

Reply:
Well, it's something that came into my mind, but this is used on another view. I decided to do it this way as it seemed a little intrusive to change other views just to make this one work. But I'm ok with both approaches.